### DISCUSSION TITLE
get_results_for_members_internal_forces - Always returns results for all members

### DISCUSSION THREAD

**Discussion by sonkaste1** [Dlubal-Software/Dlubal_CSharp_Client#41] [Q&A] [answered]
Hi there!

I have a question about the API of RFEM 6 and RSTAB 9 when using "_get_results_for_members_internal_forces_".

**The current behaviour is the following:**
_get_results_for_members_internal_forces_ always returns results for all members, even when queried for a non-existent member (e.g. no 0).

**What is expected:**
_get_results_for_members_internal_force_ returns results only for the requested member(s).

Is the behaviour correct? How to achieve the expected behaviour? 
Thank you for your answer!

Štěpán

**Comment by OndraMichal** [comment, marked as answer]
Hi @sonkaste1,
sadly, we already came across this issue and it is beeng solved under bug #106154.

**Comment by sonkaste1** [comment]
Thank you for your fast reply and the fact that you are working on a fix!
Can I track somehow the progress of #106154? I

**Comment by OndraMichal** [comment]
There is no way for you to track it but I can say that I tested the functionality today and it is fixed in our internal version.

**Reply by sonkaste1** [reply]
Thanks!

**Reply by sonkaste1** [reply]
Was the fixed released in RFEM 6.03.0002, please? I am still having the issue unfortunately with reading the internal forces.

**Reply by sonkaste1** [reply]
@OndraMichal Can you verify if this issue is fixed in release 6.03.0005, please?

**Reply by sonkaste1** [reply]
@OndraMichal @Rebecca-Coding21 
Any news about bug #106154, please?

**Reply by Rebecca-Coding21** [reply]
Hi @sonkaste1,
we are still working on fixing this bug. I can inform you here when the bug was fixed.

**Reply by Rebecca-Coding21** [reply]
Hi @sonkaste1,
the issue with get_results_for_members_internal_forces() was fixed in version RFEM 6.04.0002.

**Reply by sonkaste1** [reply]
Thank you for the fix!
